pkg/etcd/etcd_test.go

```diff
@@ -1,3 +1,219 @@
 package etcd
 
 //go:generate mockery -dir $GOPATH/src/github.com/UKHomeOffice/keto-k8/pkg/etcd -name=Clienter
+
+import (
+	"fmt"
+	"log"
+	"path"
+	"os"
+	"os/exec"
+	"testing"
+
+	"github.com/UKHomeOffice/keto-k8/pkg/fileutil"
+	"time"
+)
+
+const containerName string = "ectd_int_test"
+
+var etcdStarted = false
```

Comment:
I do agree in principle but it's a trade-off I guess but for now speed over safeguarding future poor tests.